ffi/dotnet/Devolutions.IronRdp/src/Connection.cs

```diff
@@ -122,6 +120,7 @@ private static async Task PerformCredsspSteps(ClientConnector connector, string
             var pdu = await framedSsl.ReadByHint(pduHint);
             var decoded = credsspSequence.DecodeServerMessage(pdu);
 
+            // Don't remove, DecodeServerMessage is generated, and it can return null
```

Comment:
Is the IDE triggering some message here that the code doesn't do anything? I don't know for Rider but there should be some #pragma style comment to temporarily disable the lint, it might be better if that's correct....

Reply:
Yes, the IDE reads the funtion signature, currently this is a bug that needs to be fixed in Diplomat. Since it is not a C# warning but IDE specific, #pragma will not work. The actually warning is in generated code.